golem-worker-service-base/src/repo/api_deployment.rs

```diff
@@ -54,9 +54,17 @@ impl ApiDeploymentRecord {
 
 #[async_trait]
 pub trait ApiDeploymentRepo {
-    async fn create(&self, deployments: Vec<ApiDeploymentRecord>) -> Result<(), RepoError>;
+    async fn create(
+        &self,
+        namespace: &str,
```

Comment:
What is the reason for including the namespace as a separate parameter? Looks like it's already present in the ApiDeploymentRecord

Reply:
to make sure the deployment is only to 1 namespace and its a vector of deployment record that may havve multiple namespaces, ok now it reminds me to do a validation here